gulpfile.js

```diff
@@ -28,7 +28,7 @@ const replace = require('gulp-replace');
 const jeditor = require("gulp-json-editor");
 const xmlTransformer = require("gulp-xml-transformer");
 const os = require('os');
-const git = require('gulp-git');
+const git = require('simple-git')();
```

Comment:
We usually use this require as an "import" of the library, and we "instantiate" it later when needed. You are "instantiating" it at this point. 
Are we going to go to this way now? I think is good to have all in the same style if possible.

Reply:
It seems to be the default for most packages these days that their singleton instance does not need to be instanciated, as it is returned directly by the call to `require()` - `simple-git` is an exception to this. Not sure if it is a better approach to instanciate the singleton where the package is used, which will likely be inside a function call, and risk being run multiple times if the function is called multiple times.